src/Xamarin.Android.Build.Tasks/Tasks/JavaDependencyVerification.cs

```diff
@@ -318,7 +319,7 @@ public MicrosoftNuGetPackageFinder (string? file, TaskLoggingHelper log)
 
 		try {
 			var json = File.ReadAllText (file);
-			package_list = JsonConvert.DeserializeObject<PackageListFile> (json);
+			package_list = JsonSerializer.Deserialize<PackageListFile> (json);
```

Comment:
Where this one is strongly typed, we could try the source generator? (although not sure if it works in netstandard2.0)

* https://github.com/jonathanpeppers/MAUI_CollectionView/commit/9469368e53bb31e265737e880ad755658d0f2867
* https://learn.microsoft.com/en-us/dotnet/standard/serialization/system-text-json/source-generation